templates/stats/issues.tpl

```diff
@@ -166,37 +165,37 @@
 								<template v-else>
 									{{ column.label }}
 								</template>
-							</table-column>
-						</table-header>
-						<table-body>
-							<table-row v-for="(row) in items" :key="row.key">
-								<table-cell>
+							</pkp-table-column>
+						</pkp-table-header>
+						<pkp-table-body>
+							<pkp-table-row v-for="(row) in items" :key="row.key">
+								<pkp-table-cell>
 									<a
 										:href="row.issue.publishedUrl"
 										class="pkpStats__itemLink"
 										target="_blank"
 									>
 										<span class="pkpStats__itemTitle">{{ row.issue.identification }}</span>
 									</a>
-								</table-cell>
-								<table-cell>{{ row.tocViews }}</table-cell>
-								<table-cell>{{ row.issueGalleyViews }}</table-cell>
-								<table-cell>{{ row.totalViews }}</table-cell>
-							</table-row>
+								</pkp-table-cell>
+								<pkp-table-cell>{{ row.tocViews }}</pkp-table-cell>
+								<pkp-table-cell>{{ row.issueGalleyViews }}</pkp-table-cell>
+								<pkp-table-cell>{{ row.totalViews }}</pkp-table-cell>
+							</pkp-table-row>
 							<template #no-content v-if="!items.length">
-								<table-row class="pkpStats__noRecords">
-									<table-cell :colspan="tableColumns.length" class="!py-8 !px-4 !text-center">
+								<pkp-table-row class="pkpStats__noRecords">
+									<pkp-table-cell :colspan="tableColumns.length" class="!py-8 !px-4 !text-center">
 										<template v-if="isLoadingItems">
```

Comment:
Ops..  here you can just use table-cell.

Reason why we expose the components with pkp prefix globally is so plugins can use them.

On our own pages&components we always import and register them locally. So for stats you are adding table-cell via 'components:'..   therefore no need to prefix that.

Also the no-content thingy is already supposed to insert correct row and cell with correct colspan. So you should just put there the spinner. But if its not working as expected let me know.